CHANGELOG.md

```diff
@@ -2,9 +2,12 @@
 
 All changes to `doctl` will be documented in this file.
 
-## [1.18.0] - UNRELEASED
+## [1.18.0] - 2019-05-15
 
  - #440 Add flag to set local KubeConfig's current-context. - @eddiezane
+ - #426 Add support for configuring Kubernetes maintenance windows. - @fatih
+ - #442 Add support for Kubernetes cluster upgrades. - @adamwg
+ - #443 Remove beta flag from Kubernetes commands. - @adamwg
```

Comment:
Please put these in descending numerical order 😄 

```
- #443 
- #442 
- #440 
- #426 
```

Reply:
Fixed!